docs/configuration.md

```diff
@@ -1,5 +1,21 @@
 # Configuration options
 
+### Configuration Validation
+
+k0s command-line interface has the ability to validate config syntax:
+
+```
+$ k0s validate config --config path/to/config/file
+```
+
+`validate config` sub-command can validate the following:
+
+1. YAML formatting
+2. [SANs addresses](#specapi-1)
+3. [Network providers](#specnetwork-1)
+4. [Worker profiles](#specworkerprofiles) 
+
+
```

Comment:
maybe a nitpick, but the introduction on the k0s config file, k0s.yaml, is only detailed under the Control plane section, so maybe this introduction should be made before the Configuration validation section.
I would do the following:
- Move the introduction line `k0s Control plane can be configured via a YAML config file. By default k0s server command reads a file called k0s.yaml but can be told to read any yaml file via --config option.` to the top, along with the example config file.
- Add the configuration validation section
- Change the title of "Control Plane"  (that doesn't make sense in this context) to "Configuration Keys".